### ISSUE TITLE
Spool directory not found

### ISSUE THREAD

**Issue by hisaac** [mlj/castget#44]
I'm attempting to use castget, but am having errors with my spool directory. I'd like to have the directory on a separate hard drive/volume. Here is my .castgetrc contents:

```shell
[*]
id3contenttype=Podcast
spool="/Volumes/barge/Audio/Podcast and Radio Archive"
filename=%(date)-%(title).mp3

[WireTap 2020]
url=https://www.cbc.ca/podcasting/includes/wiretap2020.xml
```

Whenever I try to run `castget`, I get this message in the terminal:

```shell
❯ castget
Updating channel WireTap 2020...
Spool directory /Volumes/barge/Audio/Podcast\ and\ Radio\ Archive not found.
```

I also tried changing the spool directory to `/home/Downloads`, but got the same type of error. Am I doing something wrong?

(I am running this on macOS Catalina 10.15.6, using the fish shell)

**Comment by mlj** [comment]
Castget uses [glib's ini file parser](https://developer.gnome.org/glib/stable/glib-Key-value-file-parser.html), which, as far as I understand, will intepret quotation marks as part of the string. So in your first example, you probably want

```
spool=/Volumes/barge/Audio/Podcast and Radio Archive
```

This does not quite fit with the error message you mention afterwards, and especially not with the fact that it failed with `/home/Downloads` where there is no spacing. Was `/home/Downloads` a mount point for a separate partition?

(I will update the documentation to mention the quotation behaviour in the ini file.)

**Comment by hisaac** [comment]
Ah good to know about the quotation marks. I will try it without the quotation marks and let you know if it works.

It's been a while since I tested this, so I don't remember exactly how I did the `/home/Downloads`. It's likely though that I didn't include the full volume path, as it was running from the volume that `/home/Downloads` was on. Would it be required to provide the full download path, e.g. `/Volumes/Macontish HD/home/Downloads`?
